fedora_messaging/tests/unit/test_cli.py

```diff
@@ -142,7 +148,7 @@ def test_no_cli_bindings(self, mock_consume):
 
     @mock.patch("fedora_messaging.cli.api.consume")
     def test_queue_and_routing_key(self, mock_consume):
-        """Assert providing improper bindings is reported."""
+        """Asser  providing improper bindings is reported."""
```

Comment:
Probably a typo here

Reply:
I noticed it while working on UT for record subcommand. Actually this whole docstr description is wrong, but I retyped it in case of "recorder" test. Since this change will be not related to this PR, I will prepare another PR to keep commits [atomic](https://bodhi.fedoraproject.org/docs/developer/index.html#contribution-guidelines).